### PR TITLE
Support RPC `findstorage`

### LINKED ISSUE #953: [Neo v3.6.0] Add new PRC `FindStorage`
https://github.com/neo-project/neo-modules/pull/805

### PR DESCRIPTION
**Pull request by mialbu**
Closes #953

### CODE REVIEW

**Comment by mialbu** [comment]
@csmuller, the integration tests are passing locally. They are failing here because the `neo3-privatenet-docker` update for Neo 3.6.0 is not yet released and the GitHub action fails to pull the image.

File: int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java [hunk 1]
```diff
@@ -621,6 +624,38 @@ public void testGetStorage() throws IOException {
...
 
+    @Test
+    public void testFindStorage() throws IOException {
+        NeoFindStorage.FoundStorage foundStorageActual = getNeow3j()
+                .findStorage(GAS_HASH, "", BigInteger.ZERO)
+                .send()
+                .getFoundStorage();
+
+        assertFalse(foundStorageActual.isTruncated());
+        assertThat(foundStorageActual.getNext(), is(BigInteger.valueOf(4)));
```
Comment (csmuller) [inline int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java:635]: @mialbu I assume you found out by experimenting that the found storage entries are truncated and that the next index is `4`. 
Do you know if the number of the entries returned in one call of this RPC method is configuratble in the node settings, e.g., like the iterator settings?

**Review by csmuller — APPROVED** [review]
💪 Great!

File: int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java (hunk 1 above)
Comment (mialbu) [reply int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java:635]: Actually, if a response is not truncated, the value `next` will directly provide you the number of existing storage entries  for the provided prefix. So, I guess that value is useful even if a response is truncated.

jiffy

Comment (csmuller) [reply int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java:635]: oh i see. many thanks for the explanation 🙏

**Pull request merged**